tests/foreman/api/test_notifications.py

```diff
@@ -231,14 +232,14 @@ def long_running_task(target_sat):
             'password': settings.server.ssh_password,
         },
     )
-    sql_date_2_days_ago = "now() - INTERVAL \'2 days\'"
+    sql_date_2_days_ago = "now() - INTERVAL '2 days'"
     result = target_sat.execute(
         "su - postgres -c \"psql foreman postgres <<EOF\n"
         "UPDATE foreman_tasks_tasks "
         f"SET start_at = {sql_date_2_days_ago}, "
         f" started_at = {sql_date_2_days_ago}, "
         f" state_updated_at = {sql_date_2_days_ago} "
-        f"WHERE id=\'{job['task']['id']}\';\nEOF\n\" "
+        f"WHERE id='{job['task']['id']}';\nEOF\n\" "
```

Comment:
is this expected in this DB query?

Reply:
Not sure, but Python sees both as the same string:
```python
[ins] In [1]: job = {'task':{'id': 123}}

[ins] In [2]: f"WHERE id=\'{job['task']['id']}\';\nEOF\n\" " == f"WHERE id='{job['task']['id']}';\nEOF\n\" "
Out[2]: True

[ins] In [3]:
```

Reply:
Same as in the other comment.

I have reverted the patch in https://github.com/SatelliteQE/robottelo/commit/9d648a751733570d4fe4b5dd3925cd240342caac and I have added a comment that will prevent formatted from touching this chunk of code.